Readability.js

```diff
@@ -2136,6 +2136,25 @@ Readability.prototype = {
           (!isList && weight < 25 && linkDensity > 0.2) ||
           (weight >= 25 && linkDensity > 0.5) ||
           ((embedCount === 1 && contentLength < 75) || embedCount > 1);
+
+        if (haveToRemove) {
+          // Check for lists to allow lists of images to remain in the page
+          if (isList) {
+            for (var x = 0; x < node.children.length; x++) {
+              if (node.children[x].localName == "li") {
+                if (node.children[x].children.length > 1) {
+                  return true;
+                }
+              } else {
+                return true;
+              }
+            }
+            li_count = node.getElementsByTagName("li").length;
+            if (img > 1 && img <= (li_count)) {
+              return false;
+            }
```

Comment:
```suggestion
              let child = node.children[x];
              if (child.localName == "li" && child.children.length) {
                  return true;
              }
            li_count = node.getElementsByTagName("li").length;
            if (img > 1 && img <= (li_count)) {
              return false;
            }
```

Reply:
I think it would be a good idea to document here why we remove some images for this specific case. It's not immediately clear to me here.